docs/src/userguide/language_basics.rst

```diff
@@ -189,48 +308,98 @@ Python functions vs. C functions
 
 There are two kinds of function definition in Cython:
 
-Python functions are defined using the def statement, as in Python. They take
+Python functions are defined using the ``def`` statement, as in Python. They take
 :term:`Python objects<Python object>` as parameters and return Python objects.
 
-C functions are defined using the new :keyword:`cdef` statement. They take
+C functions are defined using the new :keyword:`cdef` statement in Cython language or by ``@cfunc`` decorator. They take
```

Comment:
Better use tabs for this purpose instead

Reply:
Apples to other similar cases

Reply:
Closing based on following comment: https://github.com/cython/cython/pull/4242#issuecomment-879666879